src/deepqmc/fit.py

```diff
@@ -172,10 +172,6 @@ def kfac_batch_size_extractor(batch):
             value_func_has_rng=True,
             auto_register_kwargs={'graph_patterns': make_graph_patterns()},
             include_norms_in_stats=True,
-            estimation_mode='fisher_exact',
-            num_burnin_steps=0,
-            min_damping=1e-4,
```

Comment:
What happend to `min_damping`? The KFAC default seems to be 1e-8.

Reply:
I removed it since in all configurations we use a constant damping of `1e-3` so I think this min damping is never active. We can reintroduce it if we want to be extra safe.